.github/CODEOWNERS

```diff
@@ -0,0 +1,10 @@
+# Automatically request reviews when a pull request changes any owned files
+# More information: https://github.com/blog/2392-introducing-code-owners 
+
+*.groovy @dotnet/roslyn-infrastructure
+.github/ @dotnet/roslyn-infrastructure
+build/ @dotnet/roslyn-infrastructure
+samples/ @dotnet/analyzer-samples
```

Comment:
:memo: Read permission for this repository needs to be added to the @dotnet/analyzer-samples team